### ISSUE TITLE
Wont Start / Cant Delete

### ISSUE THREAD

**Issue by craigmcgowan** [leikoilja/ha-google-home#427]
I have an issue where by this integration wont start. 

My error logs show

`
2021-12-29 10:54:53 ERROR (MainThread) [homeassistant.setup] Setup failed for google_home: Unable to import component: cannot import name 'Device' from 'glocaltokens.client' (/usr/local/lib/python3.9/site-packages/glocaltokens/client.py)
`

However, I am also unable to delete the integration from my system to try and reinstall it. When i try this, I get

`
2021-12-29 11:03:34 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
File "/usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
resp = await request_handler(request)
File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 504, in _handle
resp = await handler(request)
File "/usr/local/lib/python3.9/site-packages/aiohttp/web_middlewares.py", line 117, in impl
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 60, in security_filter_middleware
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 220, in forwarded_middleware
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 28, in request_context_middleware
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 78, in ban_middleware
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 181, in auth_middleware
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 137, in handle
result = await result
File "/usr/src/homeassistant/homeassistant/components/config/config_entries.py", line 71, in delete
result = await hass.config_entries.async_remove(entry_id)
File "/usr/src/homeassistant/homeassistant/config_entries.py", line 831, in async_remove
await entry.async_remove(self.hass)
File "/usr/src/homeassistant/homeassistant/config_entries.py", line 487, in async_remove
component = integration.get_component()
File "/usr/src/homeassistant/homeassistant/loader.py", line 522, in get_component
cache[self.domain] = importlib.import_module(self.pkg_path)
File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
return _bootstrap._gcd_import(name[level:], package, level)
File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
File "<frozen importlib._bootstrap_external>", line 850, in exec_module
File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
File "/config/custom_components/google_home/__init__.py", line 17, in <module>
from .api import GlocaltokensApiClient
File "/config/custom_components/google_home/api.py", line 11, in <module>
from glocaltokens.client import Device, GLocalAuthenticationTokens
ImportError: cannot import name 'Device' from 'glocaltokens.client' (/usr/local/lib/python3.9/site-packages/glocaltokens/client.py)
`

Im running HA 2021.12.6

Any ideas what the issue is?

Thaks

**Comment by DurgNomis-drol** [comment]
It looks like the python module `glocaltokens` is not installed. Can you try to install it manually through the terminal. Run command `pip3 install glocaltokens` :smile:

**Comment by craigmcgowan** [comment]
I have installed glocaltokens on my machine but still get the same issue.

I am running Home Assistant Supervised as a docker container and have installed glocaltokens on my actual Debian machine. Is this correct or do I need to install it within the HA container?

Many thanks for your help

**Comment by DurgNomis-drol** [comment]
You need to install it in the HA docker container 😊

**Comment by craigmcgowan** [comment]
Okay, so if i remember correctly, "Terminal & SSH" gets me into the HA docker container. But pip3 isnt available as python isnt even installed. Am I in the right place,please?

Furthermore I shoul say that when I installed ha-google-home last night everything was working fine and I had all the sensors from this integration which makes me unsure that a missing dependency is the issue??

Kind regards

**Comment by craigmcgowan** [comment]
Following on from my previous message where I said it worked last night... Ive just restored to by 1am backup and its working again now.
I was also able to delete the integration, restart the server, reinstall the integration and it also worked. A reboot and everything and its still working.

I did notice that I must have updated to 2021.12.6 after 1pm last night as im back on 2021.12.5 after my restore so I wonder if there was a breaking change in between them two?

**Comment by DurgNomis-drol** [comment]
It sounds really strange, did you change something in you installation?

**Comment by craigmcgowan** [comment]
No, I havent made any changes. It was working fine last night at 1am when I was fiddling before going to sleep. For some reason HA restarted itself at about 5 ad 8am this morning and then when I looked at it about 8.15am I was getting the above error messages.
All I did after 1am was upgrade to 21.12.6 and it wasnt working this morning.
Rolling back to 1am has fixed my issue (fingers crossed) and ive since upgraded again to .6 and its still working.

Thank you for your help.
